### DISCUSSION TITLE
Suggestions!

### DISCUSSION THREAD

**Discussion by yagamiyakumo** [mienaiyami/yomikiru#304] [Ideas]
The app is great but I think a couple of things that may be neat if possible:

1. Setting options to open only 1 instance at any time. Nested setting to offer to bookmark current chapter before a different manga is opened, can be turn off
2. Keep the image location unchanged when chapter list is pinned/opened. (Similar to how reader settings currently works). Currently the manga will shift to the right when chapter list is pinned/opened
3. Setting options to retain chapter list & reader settings pinned/open state even after closing Yomikiru
4. X.Y chapters should follow after X chapters. Eg Chapter 10 should come before Chapter 10.5
5. Port it to Steam Deck (I think it uses KDE?) please? Would be great if it's on discover store! >_<

**Comment by mienaiyami** [comment]
> 2\. Keep the image location unchanged when chapter list is pinned/opened. (Similar to how reader settings currently works). Currently the manga will shift to the right when chapter list is pinned/opened

If you just hover over side list it will stay open without shifting layout. Maybe I didnt understand your request, but it will only shift images to right after you click on the pin button.



> * X.Y chapters should follow after X chapters. Eg Chapter 10 should come before Chapter 10.5

Im sure thats how it works currently. Check your sorting method.



> 5\. Port it to Steam Deck (I think it uses KDE?) please? Would be great if it's on discover store! >_<

Actually, I cant put it on any story because I cant afford the code signing certificate. 

---

Please make suggestions and request using the issue feature (blank issue), as it is easier to manage and track compared to discussions.

**Reply by yagamiyakumo** [reply]
Sorry, still pretty new to github. I'll create it over there later

**Comment by GladTech** [comment]
Hello, I have been using Yomikiru and I really like it so far. It has a clean interface, pretty easy to use, and I love the smooth vertical scrolling. 
That said, I have a suggestion that is pretty rare and non-existent amongst the other readers and that's the ability to auto scroll. 
I would absolutely LOVE such a feature. 
I have seen one or 2 browser extensions that offer such a feature, but have yet to find a standalone desktop app that offers the same functionality. 
[Hidari - Manga & Webtoon Reading Assistant](https://chromewebstore.google.com/detail/bomhjipgphmpallgkkdjbabpegdaolmd) is one of them and the other is a [youtube video](https://www.youtube.com/watch?v=J-AA_UEx1A8) where he created a python script or extension that did it as well.
Anyway, it's something I think is a good idea, so I figured I'd mention it to you and see what you thought. 
Thank you so much for the app and I appreciate all your hard work and effort.

**Reply by mienaiyami** [reply]
> [Hidari - Manga & Webtoon Reading Assistant](https://chromewebstore.google.com/detail/bomhjipgphmpallgkkdjbabpegdaolmd)

You can already do that with middle mouse button. Just click middle mouse button, release it(without moving mouse), and now move mouse in any direction to set speed.

And as for the video you linked, I had tried such method in past, it only sound great in theory but when in real use. Time per page is not just dependent on the word count, the images can also take a lot of time to understand. I know many manga which have many continuous pages with no text at all, but still, it takes variable time per page.

I'm aware of the scrolling with hand issue but Yomikiru provides 2 different keyboard scroll shortcuts, both with separate speed (configurable) which keyboard. Maybe I will add a clickable button to do same function as middle mouse button in future.

**Reply by GladTech** [reply]
Ok, thanks for the insight and for letting me know about the middle mouse button!
It works for my purposes, so I'm glad to hear about it. 
And yeah, the timer isn't going to work for 100% of scenarios, but I think it works in most cases. 
That being said, the middle button works just as well and super easy to use. 
Loving the app and thanks again!

**Comment by aldotessaiga** [comment]
First of all, thank you for the excellent program, but please add an option so that the program does not open maximized!! It is very bad that the program always opens maximized and does not remember its position on the screen, please enable a way for it to remember its last position on the screen or for the user to be able to configure the program's opening position.
